src/kandidat/kandidatside/komponenter/header/Kandidatheader.tsx

```diff
@@ -15,21 +14,24 @@ type Props = {
 const Kandidatheader = ({ kandidatnavigering, kandidatnr, brødsmulesti }: Props) => {
     useMaskerFødselsnumre();
 
-    const kandidat = useKandidat(kandidatnr);
+    const { kandidatsammendrag, error, isLoading } = useKandidatsammendrag(kandidatnr);
 
-    const brødsmulestiMedNavn =
-        kandidat.kind === Nettstatus.Suksess
-            ? [
-                  ...(brødsmulesti ?? []),
-                  {
-                      tekst: formaterNavn(kandidat.data),
-                  },
-              ]
-            : brødsmulesti;
+    if (isLoading) {
+        return <></>;
+    }
+
+    const brødsmulestiMedNavn = error
```

Comment:
I perioden mellom at HTTP-requesten for kandidatsammendraget er sendt og at HTTP-responsen er returnert, så vil både `error` og `kandidatsammendrag` være `undefined`, og derfor sendes `undefined` inn i `formaterNavn`.  Så tenker den gamle definisjonen av variabelen burde brukes, men med `kandidatsammendrag` i stede for `kandidat.kind === Nettstatus.Suksess` i sjekken.